doc/development-context-aware-applications/deep-dive.md

```diff
@@ -1,12 +1,5 @@
 Should you wish to learn much more about this, please check: 
 
    - [Introduction to NGSI API and Orion Context Broker](http://bit.ly/fiware-orion)
-   - [Github repository (reference implementation) ](https://github.com/Fiware/context.Orion)
-   - [Play with the Docker container](https://hub.docker.com/r/fiware/orion/)
-   - [Browse NGSI v1](http://fiware.github.io/context.Orion/api/v1/)
-   - [Browse NGSI v2](http://fiware.github.io/specifications/ngsiv2/latest/)
```

Comment:
Hi again @jmcanterafonseca 

This content are in introduction section. You can see the both full API NGSI, and the OCB. About Docker installation, if you explore the github NGSI site you can find it and use it. 

Kind regards